apps/src/templates/projects/submitProjectDialog/submitProjectApi.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * TODO: Sends a post request to submit the project.
+ */
+export async function submitProject(submissionDescription: string) {
+  console.log('submitProject', submissionDescription);
+}
+
+/**
+ * TODO: Sends a get request to submit the project.
+ */
+export function getSubmissionStatus() {
```

Comment:
super nit: i know this is still todo but presumably this would be `async` as well?

Reply:
yes - will be `async`.